FindingYourData.xml

```diff
@@ -0,0 +1,43 @@
+Enter file contents here<?xml version="1.0" encoding="UTF-8"?>
+<cheatsheet title="Generating links to existing data stored on your machine"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="..\contentFile.xsd">
+	<intro>
+		<description>As mentioned previously, Debrief NG stores its data in a local <i>Project</i>
+			folder. While these are frequently created from scratch to store new analysis data, you
+			can also set your existing data directories as sub-folders (via links).<br/>The
+			following procedure guides you through generating links to your existing data folders.
+			However, if the <b>Create Project</b> form opened when the application started and you
+			provided both a project name and indicated that you would like sample data to be
+			imported, then you can skip the following steps and move on to the next cheat sheet,
+				<b>Viewing some data</b>.</description>
+	</intro>
+	<item skip="true" title="Open Navigator if it is closed">
+		<description>Open <b>Navigator</b> if it is closed by using the <b>Ctrl+3</b> shortcut and
+			then either clicking on the option in <b>Previous Choices</b> or typing
+			&apos;navigator&apos; in the search field and clicking on it when it appears.
+		</description>
+	</item>
+	<item skip="true" title="Check you have a workspace">
+		<description>Before you can link existing data, you need to define a project for your work:
+			a <i>project</i> is a parent folder which stores your links. If your <b>Navigator</b> window is empty,
+			you must first complete the <b>Generating a project for your data</b> cheat-sheet (above).
+		</description>
+	</item>
+	<item skip="true" title="Add a new Folder">
+		<description>
+			To add a new folder:
+			<br/><b>1.</b> Right-click on your current project in the <b>Navigator</b> view, the <b>New</b> sub-menu will show.
+			<br/><b>2.</b> Select <b>Folder</b> and the <b>New Folder</b> wizard will open.
+			<br/>Here though, instead of creating a fresh child folder to store our data, we're going to link to an existing folder.
```

Comment:
@alastrina123 - I can see you've identified lots of *&apos;* instances - but you have still missed a few instances - I think I counted 9.